doc/modules/changes/20190313_gassmoeller

```diff
@@ -0,0 +1,8 @@
+New: There is now an option to output visualization data as higher order
+polynomials. This is an improvement in accuracy and requires less disk space
+than the 'Interpolate output' option that was available before (the new option
+requires the old option to be set, which it is by default). However the new
```

Comment:
What is the reason you choose to make this option dependent on an other option? Can't we just deal with this internally?

Reply:
We could, but then setting `Interpolate output` would not change anything if `Write higher order output` is set. That might be confusing as well. Would you prefer the internal solution?

Reply:
I Ithink the best option would be to have the user choose of the three options in one parameter, but that would not be backwards compatible. 

I would personally prefer to have this done internally, because it is one thing less for the user to worry about/check, but I do not feel strongly about it and am also fine with leaving it this way. I will leave it up to you.

Reply:
In that case I prefer the current way :smile:. In a few years the question will become  irrelevant anyway, as we can then simply always use the higher order output if the interpolate option is set to true.
Maybe leave some time for others to comment (e.g. until tomorrow or so) and then feel free to merge.